frontend/src/api/instances.js

```diff
@@ -127,10 +127,18 @@ const getInstanceDevices = async (instanceId, cursor, limit) => {
 const getInstanceDeviceSettings = async (instanceId) => {
     return client.get(`/api/v1/projects/${instanceId}/devices/settings`).then(res => res.data)
 }
+
 const updateInstanceDeviceSettings = async (instanceId, settings) => {
     return client.post(`/api/v1/projects/${instanceId}/devices/settings`, settings).then(res => res.data)
 }
 
+const rollbackInstance = async (projectId, snapshotId) => {
```

Comment:
/s/projectId/instanceId

Reply:
sorry @pez, I dont understand?Hi Pez,  this fix works locally and follows suit with all the other functions in the file.

I realise this should be (will be eventually renamed)

Reply:
> follows suit with all the other functions in the file.

No it doesn't. All other functions use `instanceId`.

Reply:
sorry, I meant I followed suit with the source call

https://github.com/flowforge/flowforge/blob/2cfeb77378110f8297e6cb603c501f5d4642be6f/frontend/src/api/projectSnapshots.js#L27

and other functions

https://github.com/flowforge/flowforge/blob/2cfeb77378110f8297e6cb603c501f5d4642be6f/frontend/src/api/projectSnapshots.js#L10

Reply:
I have also renamed this in the source function call too in commit [a8eb848](https://github.com/flowforge/flowforge/pull/2030/commits/a8eb8487bd5f8027966ef3b64a4dc2d9fe01f4a4)